HorizonCore/src/main/java/com/hubspot/horizon/HttpRequest.java

```diff
@@ -135,13 +135,22 @@ public static class Options {
 
     public static Options DEFAULT = new Options();
 
-    private Optional<Integer> maxRetries = Optional.absent();
-    private Optional<Integer> initialRetryBackoffSeconds = Optional.absent();
-    private Optional<Integer> maxRetryBackoffSeconds = Optional.absent();
-    private Optional<RetryStrategy> retryStrategy = Optional.absent();
+    private Optional<Integer> requestTimeoutSeconds = Optional.empty();
```

Comment:
Is adding this at second precision mainly to keep things consistent. Having millis (or even a Duration) here would be more flexible, though I don't know if people ever really need sub-second timeouts.

Reply:
Yeah mostly just for consistency since everything is currently at second granularity. Swapping to duration would definitely be an improvement, but maybe we should do that across the board in a separate PR

Reply:
As long as you don't think that only having support for second-precision will unnecessarily limit something, that sounds good to me.

Reply:
My use-case is for increasing timeouts, so should be fine 😄